src/main/java/anichan/Main.java

```diff
@@ -103,7 +103,7 @@ public Main() {
 
         // ========================== Workspace Setup ==========================
 
-        LOGGER.log(Level.INFO, "Workspace setup... ");
+        LOGGER.log(Level.INFO, "Workspace setup..");
```

Comment:
Perhaps we could standardize the amount of periods we use for logs, and when to use them?

Reply:
Yep, I think we could standardize to 2 periods, ".."?
I think we could use it during the initialization process? Seems like the only place it would fit.

Reply:
Alright sounds good!